packages/terra-button-group/tests/jest/ButtonGroup.test.jsx

```diff
@@ -4,6 +4,11 @@ import ButtonGroup from '../../src/ButtonGroup';
 const button1 = <ButtonGroup.Button key="1" />;
 const button2 = <ButtonGroup.Button key="2" />;
 
+it('should render an empty component', () => {
```

Comment:
I removed a test nightwatch was doing with an empty button group and moved it into jest as its more effective at determining if things don't need to render.